utils/fetch_languages.py

```diff
@@ -174,14 +174,17 @@ def write_languages_file(languages):
                    + '# this file is generated automatically by utils/update_search_languages.py\n'\
                    + '\nlanguage_codes = ('
     for code in sorted(languages):
-        file_content += '\n    (u"' + code + '"'\
-                        + ', u"' + languages[code]['name'].split(' (')[0] + '"'\
-                        + ', u"' + languages[code].get('country', '') + '"'\
-                        + ', u"' + languages[code].get('english_name', '').split(' (')[0] + '"),'
+        if 'name' in languages[code]:
```

Comment:
There is an additional check here, otherwise the script crash on this entry:
```python
{'counter': ['qwant', 'qwant images', 'qwant news', 'qwant social'], 'country': ''}
```